genkit-tools/common/package.json

```diff
@@ -63,61 +64,61 @@
   },
   "author": "genkit",
   "license": "Apache-2.0",
-  "types": "./lib/types/types/index.d.ts",
+  "types": "./lib/types/src/types/index.d.ts",
```

Comment:
this is required as the package.json is included in the build now, which lives above `src`